core/src/main/java/org/apache/carbondata/core/datastore/impl/FileFactory.java

```diff
@@ -365,10 +366,11 @@ public static boolean createNewLockFile(String filePath, FileType fileType) thro
   public static String getUpdatedFilePath(String filePath, FileType fileType) {
     switch (fileType) {
       case HDFS:
-      case ALLUXIO:
       case VIEWFS:
       case S3:
         return filePath;
+      case ALLUXIO:
+        return StringUtils.containsAny(filePath,"alluxio")?filePath: "alluxio://"+filePath;
```

Comment:
Why is this special handling for alluxio?

Reply:
Alluxio does the path translation in the FileSystem , that s why we have this issue. https://github.com/Alluxio/alluxio/blob/master/core/client/hdfs/src/main/java/alluxio/hadoop/AbstractFileSystem.java